action/claimreward.go

```diff
@@ -217,8 +232,16 @@ func NewClaimFromRewardingFundFromABIBinary(data []byte) (*ClaimFromRewardingFun
 	if ac.data, ok = paramsMap["data"].([]byte); !ok {
 		return nil, errDecodeFailure
 	}
-	if ac.address, ok = paramsMap["address"].(string); !ok {
+	var s string
+	if s, ok = paramsMap["address"].(string); !ok {
```

Comment:
should keep backword compatibility, allowing without address param

Reply:
add a 2nd commit to keep both v1 and v2 ABI